website/cue/reference/components/sources/kubernetes_logs.cue

```diff
@@ -42,6 +42,11 @@ components: sources: kubernetes_logs: {
 			"""
 				[Kubernetes](\(urls.kubernetes)) version `\(services.kubernetes.versions)` is required.
 				""",
```

Comment:
Just noting that the rendering seems to smush the two warnings together.

Reply:
Without editing the formatting, I could move the version message over to the "notices" section, leaving this alone. I *think* that should make the message standout more.

Reply:
Nah, I don't think it's an issue - mostly observation. I can't think of a case where we'd want two side-by-side warning components. Just the thought of that visual feels weird.